src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.MovieSessionRequestDto;
+import mate.academy.spring.dto.MovieSessionResponseDto;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.mapping.MovieSessionDtoMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    @Autowired
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    @PostMapping
+    public MovieSessionResponseDto create(@RequestBody MovieSessionRequestDto requestDto) {
+        MovieSession movieSession = movieSessionService
+                .add(movieSessionDtoMapper.parseToModel(requestDto));
+        return movieSessionDtoMapper.parseToDto(movieSession);
+    }
+
+    @GetMapping(value = "/available")
+    public List<MovieSessionResponseDto>
+            get(@RequestParam Long movieId,
+                @RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date) {

```

Comment:
```suggestion
    public List<MovieSessionResponseDto> get(@RequestParam Long movieId,
    @RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date) {
```
never move the method name to a new line